Gemfile

```diff
@@ -32,6 +37,8 @@ gem "jbuilder", "~> 2.5"
 # Reduces boot times through caching; required in config/boot.rb
 gem "bootsnap", ">= 1.1.0", require: false
 
+gem "naught"
```

Comment:
I haven't heard of this gem.

Reply:
Avdi talks about this concept in his Confident Ruby book. I forget if he mentions the gem explicitly. I don't actually find much use for it other than non-authenticated user states, but it is possible I just haven't looked hard enough. 

For a single use case, you could just make your own custom class that defined all the null behavior.